tests/net/socket/tcp/src/main.c

```diff
@@ -31,21 +31,21 @@ static void test_bind(int sock, struct sockaddr *addr, socklen_t addrlen)
 {
```

Comment:
Please place all the test related code to commit that creates the tests, now there is test changes in the 1st commit for example.